### PR TITLE
docker-compose update

### PR DESCRIPTION
**Pull request by soorajoasis**
The updated version of docker compose solution for dx. 

Changes.

The new DAM database system is configured.
Introduced nginx reverse proxy.

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        # set client body size to 500M #
+        client_max_body_size 500M;
+
+        keepalive_timeout  65;
+
+        #gzip  on;
+
+        server {
+                listen          80;
+                server_name     localhost;
```
Comment (timpeteler) [inline nginx.conf:35]: this shouldn't just be localhost
image

File: nginx.conf [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+                # DAM API
+                location /dx/api/dam {
+                        proxy_set_header Host $host;
+                        proxy_pass  http://dx-dam:3001;
+                }
+
+                # DAM UI
```
Comment (timpeteler) [inline nginx.conf:43]: duplication

File: nginx.conf [hunk 3]
```diff
@@ -0,0 +1,78 @@
+user  nginx;
```
Comment (timpeteler) [inline nginx.conf:1]: I guess you just copied the nginx.conf from the PR check env, right?
I'd ask you to distill it down to the bare minimum of what is needed (maybe just the proxy rules?)

**Review by timpeteler — CHANGES_REQUESTED** [review]

**Review by timpeteler — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -302,6 +302,20 @@ cd ./local-docker-compose
 installApps.bat
 ```
 
+### Connecting to your DX and applications.
+
+To access your dx environment, navigate to *http://<PORTAL_HOST>/wps/portal*
+
+Example: http://localhost/wps/portal
```
[the remark this answers is not in the document]
Comment (soorajoasis) [reply README.md:306]: Done.

**Review by timpeteler — APPROVED** [review]

**Pull request merged**